providers/cloudflare/cloudflareProvider.go

```diff
@@ -193,6 +235,20 @@ func (c *CloudflareApi) preprocessConfig(dc *models.DomainConfig) error {
 				rec.Metadata[metaProxy] = val
 			}
 		}
+		// CF_REDIRECT record types. Encode target as $FROM,$TO,$PRIO,$CODE
+		if rec.Type == "CF_REDIRECT" || rec.Type == "CF_TEMP_REDIRECT" {
+			parts := strings.Split(rec.Target, ",")
+			if len(parts) != 2 {
+				return fmt.Errorf("Invalid data specified for cloudflare redirect record")
```

Comment:
Alternatively, encode the string as follows:

    DELIMITER + src + DELIMITER + dst

Set DELIMITER to comma. If comma is in src or dst, try other chars.